contentcuration/contentcuration/models.py

```diff
@@ -975,6 +978,13 @@ class Channel(models.Model):
         verbose_name="languages",
         blank=True,
     )
+    version_info = models.OneToOneField(
+        "ChannelVersion",
+        related_name="channel_version_info",
```

Comment:
As a related name, we can just use `channel` so that if we have a `version_info` object, we can access the channel through `version_info.channel.`